src/komponenter/aap/aap.tsx

```diff
@@ -1,7 +1,9 @@
 import * as React from 'react';
-import Lenke from 'nav-frontend-lenker';
+// import Lenke from 'nav-frontend-lenker';
 import * as queryString from 'query-string';
-import { HoyreChevron } from 'nav-frontend-chevron';
+// import { HoyreChevron } from 'nav-frontend-chevron';
```

Comment:
Kan vi fjerne de to utkommenterte importene helt? 🙂

Reply:
seff